raiden/storage/sqlite.py

```diff
@@ -579,16 +579,19 @@ def get_statechanges_records_by_range(
             for entry in cursor
         ]
 
-    def _query_events(self, limit: int = None, offset: int = None) -> List[Tuple[str, datetime]]:
-        limit, offset = _sanitize_limit_and_offset(limit, offset)
-        cursor = self.conn.cursor()
-
-        cursor.execute(
-            """
-            SELECT data, timestamp FROM state_events
-                ORDER BY identifier ASC LIMIT ? OFFSET ?
-            """,
-            (limit, offset),
+    def _query_events(
+        self,
+        limit: int = None,
```

Comment:
Shouldn't limit/offset be marked as Optional?

Reply:
`= None`  marks it as optional for mypy. 

And at least according to our `Contributing.MD`: https://github.com/raiden-network/raiden/blob/develop/CONTRIBUTING.md

We omit the `Optional[]` in this case.